akka-actor-tests/build.sbt

```diff
@@ -7,3 +7,5 @@ Formatting.formatSettings
 publishArtifact in Compile := false
 
 Dependencies.actorTests
+
+AkkaBuild.dontPublishSettings
```

Comment:
Have you tried `disablePlugins(akka.Publish)` instead? This is an autoplugin that is added to every project. I think disabling this plugin on a project should disable publishing.